<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_pass_form</class>
 <widget class="QWidget" name="new_pass_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_cod">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
font: 700 italic 11pt &quot;Segoe UI&quot;;
border: 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_continue">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>480</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border: 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
font: 700 italic 11pt &quot;Segoe UI&quot;;
border: 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
font: 700 italic 11pt &quot;Segoe UI&quot;;
border: 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>620</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border: 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>720</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/dom/image/36.png);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_showPassword">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>290</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border: 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>graphicsView2</zorder>
  <zorder>lineEdit_cod</zorder>
  <zorder>pushButton_continue</zorder>
  <zorder>lineEdit</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>pushButton_exit</zorder>
  <zorder>pushButton_showPassword</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
